### PR TITLE
MNTOR-2013 - Exposure Card Refinements

### PR DESCRIPTION
**Pull request by codemist**
# References: 
Jira: MNTOR-2013

# Description
This PR resolves a couple of exposure card refinements.
 - Adds extra padding to the expanded details in the exposure card
 - Increase width of the Fix it Button
 - Disables "let's keep going" CTAs for Foxfooding
 - Update counter for data classes, removing "_ found" after each data class
 

# Screenshot (if applicable)

image
image

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Review by flodolo — APPROVED** [review]

File: src/app/components/client/ExposureCard.tsx [hunk 1]
```diff
@@ -321,15 +322,9 @@ const SubscriberBreachCard = (props: SubscriberBreachCardProps) => {
       <div className={styles.detailsFoundItem}>
         <dt>
           <span className={styles.exposureTypeIcon}>{props.icon}</span>
-          {props.exposureCategoryLabel}
+          {props.exposureCategoryLabel}: {props.count}
```
Comment (pdehaan) [inline src/app/components/client/ExposureCard.tsx:325]: @flodolo Does this work? Or do we need to let localizers control the order of `label: count`?

Comment (flodolo) [reply src/app/components/client/ExposureCard.tsx:325]: It's not the order, it's the spaces. This needs to be exposed as a localizable string.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: locales-pending/premium.ftl [hunk 2]
```diff
@@ -149,6 +143,10 @@ exposure-card-label-exposure-type = Exposure type
 exposure-card-label-date-found = Date found
 # Status of the exposure card, could be In Progress, Fixed or Action Needed
 exposure-card-label-status = Status
+# Variables: 
+# $category_label is the data breach exposure type that was leaked. Eg. Email, IP Address.
+# $count is the number of times that the data type was leaked.
+exposure-card-label-and-count = { $category_label }: { $count }
```
Comment (flodolo) [inline locales-pending/premium.ftl:149]: Are we using the localized category name as is? If so, we have a capitalization problem that I raised before.

Not blocking for this bug, but something to keep in mind as a follow-up. Those categories were created to be used in a list, not as stand-alone items, and that's a problem that we'll keep having and should be approached consistently.

**Review by flodolo — APPROVED** [review]

**Review by Vinnl — COMMENTED** [review]
Had a couple of questions.

File: src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx [hunk 3]
```diff
@@ -150,6 +150,7 @@ export const DashboardTopBanner = (props: DashboardTopBannerProps) => {
             // do something
           }}
           small
+          disabled
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx:153]: Ah gotcha, this is just the CTA that brings users to breach resolution. I think I'm going to run into conflicts with this in my routing PR and then hopefully remember to disable it in the right time, but I'll figure that out then :shrug:

**Pull request merged**